File: cs50/Psets/pset6/sentimental-mario-more/mario.py
from cs50 import get_int

altura = 0

while altura < 1 or altura > 8:

    altura = get_int("Ingrese la altura deseada: ")

for i in range(altura):
    for j in range(altura - i - 1):
        print(" ", end="")

    for s in range(i + 1):
        print("#", end="")

    print("  ", end="")

    for s in range(i + 1):
        print("#", end="")

    print("\n", end="")

File: cs50/Psets/pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void triangulo();
void ejecutar();

int main()
{

    ejecutar();
    return 0;
}

void triangulo()
{
    int altura;
    do
    {
        printf("---------------------------------------\n");
        altura = get_int("ingrese la altura: ");
    }
    while (altura < 1 || altura > 8);

    for (int i = 1; i <= altura; i++)
    {
        for (int j = 1; j <= altura - i; j++)
        {
            printf(" ");
        }
        for (int k = 1; k <= i; k++)
        {
            printf("#");
        }
        for (int n = 1; n <= 2; n++)
        {
            printf(" ");
        }
        for (int l = 1; l <= i; l++)
        {
            printf("#");
        }
        printf("\n");
    }
}

void ejecutar()
{

    do
    {
        triangulo();
        break;
    }
    while (!0);
}
